portfolio/pom.xml

```diff
@@ -12,7 +12,8 @@
     <maven.compiler.source>11</maven.compiler.source>
```

Comment:
No need to change this time, but in the future a more descriptive commit message is very helpful for debugging your code later. For example, the commit in the main software sprint repo said "Bump jetty.version from 9.4.31.v20200723 to 11.0.0 in /portfolio", which was really helpful in narrowing down the problem. Commit messages which describe the code change you made instead of the file you changed make your life way easier in the future.